clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -897,6 +906,10 @@ public void run() {
       storeSetting = getSettingsFromController(controllerClient, pushJobSetting);
       inputStorageQuotaTracker = new InputStorageQuotaTracker(storeSetting.storeStorageQuota);
 
+      if (pushJobSetting.repushTtlInHours != -1 && storeSetting.isChunkingEnabled) {
+        throw new VeniceException("Chunking is not supported when using repush with ttl feature");
```

Comment:
I think the message should be another way around, right? 

Repush TTL is not supported when the store has chunking enabled.

Reply:
You are right!